### DISCUSSION TITLE
+/-value in addition to set hunger.

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#8134] [Enhancement]
I want a command that adjusts value up/down rather than sets it. If it's not too difficult, please add it. :)

**Comment by CloudeLecaw** [comment]
i love your idea and total agree :D With current hunger command we can set only fixed values, butbsadly not adjust what would be nice specialy when we create own food with attached commands. The same counts for saturation.

it would be nice to be able to use +/- as an option
to the current one to adjust indtead of just setting.
For hunger it could be:
`cmi hunger (playerName) [(+|-)amount] (-s)`
For saturation it could be:
`cmi saturation (playerName) [(+|-)amount] (-s)`
(well i know the end point of hunger is 0 and the end point of saturation is 0 too, to it should not be possible to be lower then 0 even if you reduce a rest of 3 by 20)

also then we dont need to use the placeholders anymore ro adjust food and saturation.
currently for adjust food and saturation we use
`%cmi_equationint_{player_food_level}+1%`
and
`%cmi_equationint_{player_saturation}+1%`
on the attached commands we attached to the
own made food. (in case of adjust +1)

This would allow us to adjust and also set hunger and saturation much easier than it is for now with the placeholders :3   

well for saturation pls also add the useage of -s :)
(i dont know if its still missing or just not on the wiki...)

**Comment by CloudeLecaw** [comment]
by the way.. i noticed the heal command also works with that structure.
`cmi heal (playerName/all) ((+|-)healamount/healpercent) (-nofeed) (-ignoreffects) (-dontextinguish)`... also there i miss the -s :3
